### DISCUSSION TITLE
Baggage propagation in java with boot 3.0

### DISCUSSION THREAD

**Discussion by VinishKumar1** [open-telemetry/community#1351] [Community Discussions]
Hello,

We were building a spring reactive application and were trying to use open-telemetry java-agent to propagate across web-client and kafka. We needed some help for understanding how can we send baggage (like correlation-id) using the same, can someone please help?

Thanks,
Vinish

**Comment by trask** [comment]
hi @VinishKumar1! I'd recommend asking in [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java) or [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation)
